test/service_instance_and_binding_test/service_binding_test/service_binding_test.go

```diff
@@ -239,6 +239,90 @@ var _ = DescribeTestsFor(TestCase{
 				ctx.CleanupAdditionalResources()
 			})
 
+			Describe("Get Parameters, service does not exist", func() {
```

Comment:
- usually  "when" is used to tell what exists before each it, describe is used to introduce the topic under test.
We can structure retrievable service instance tests under the GET describe or add a new describe block  as Describe("retrievable service instance parameters")

- The sentence inside the brackets should not be capitalized since when it's printed it will already have the capital from: Describe, When, Context accordingly.